interpro7dw/interpro/ftp/iprscan.py

```diff
@@ -236,10 +281,19 @@ def _export_entries(cur: oracledb.Cursor, entries_file: Path):
         }
 
     with entries_file.open("wt") as fh:
-        json.dump({
-            "databases": {n: v for _, n, v in databases.values()},
-            "entries": entries
-        }, fh)
+        json.dump(entries, fh)
+
+    databases = {n: v for _, n, v in databases.values()}
+    databases["InterPro"] = ipr_version
+    with database_file.open("wt") as fh:
+        json.dump(databases, fh)
+
+
+def build_member_archive(member: str, version: str, outdir: str, data_dir: str,
+                        pkg_func: Callable[[Path, str, tarfile.TarFile], None]):
+    member_output = Path(outdir) / f"{member}-{version}.tar.gz"
```

Comment:
```suggestion
def build_member_archive(member: str, version: str, outdir: Path, data_dir: Path,
                        pkg_func: Callable[[Path, str, tarfile.TarFile], None]):
    member_output = outdir / f"{member}-{version}.tar.gz"
```

`outdir` and `data_dir` are of type `Path`:

https://github.com/ProteinsWebTeam/interpro7-dw/blob/ace9f7252920a2112e48d9fc4bd9ec75a471a8c9/interpro7dw/interpro/ftp/iprscan.py#L17

https://github.com/ProteinsWebTeam/interpro7-dw/blob/ace9f7252920a2112e48d9fc4bd9ec75a471a8c9/interpro7dw/interpro/ftp/iprscan.py#L45